src/Silex/Provider/SecurityServiceProvider.php

```diff
@@ -592,6 +596,43 @@ public function boot(Application $app)
         }
     }
 
+    /**
+     * @param ContainerInterface $container
+     *
+     * @return void
+     */
+    protected function addRouter(ContainerInterface $container)
+    {
+        if (!$container->has('routers')) {
+            return;
+        }
+
+        $loader = new ClosureLoader();
+        $fakeRoutes = $this->fakeRoutes;
+
+        $resource = function () use ($fakeRoutes) {
+            $routeCollection = new RouteCollection();
+            foreach ($fakeRoutes as $route) {
+                list($method, $pattern, $name) = $route;
+
+                $route = new Route($pattern);
+
+                $controller = function (Request $request) {
+                    throw new \LogicException(sprintf('The "%s" route must have code to run when it matches.', $request->attributes->get('_route')));
```

Comment:
I guess it should be in use statements

Reply:
@dereuromark do we have a sniff for this?

Reply:
Pretty sure we do, but seems to not be activated for this repo.